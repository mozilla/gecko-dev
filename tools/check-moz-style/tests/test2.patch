# Test
diff --git a/tests/test2.cpp b/tests/test2.cpp
new file mode 100644
--- /dev/null
+++ b/tests/test2.cpp
@@ -0,0 +1,3 @@
+int main() {
+  return 0;
+}
