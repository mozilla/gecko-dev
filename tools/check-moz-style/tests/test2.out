patch:0:  Patch does not have a summary.  [patch/nosummary] [3]
patch:0:  Patch does not have a description.  [patch/nodescription] [3]
tests/test2.cpp:1:  No copyright message found.  [legal/copyright] [3]
Done processing tests/test2.cpp
